src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -163,16 +165,17 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
       .pipe(takeUntil(this.destroy$))
       .subscribe((planState) => {
         this.plan$.next(planState.all[planState.currentPlanId!]);
-        this.scenarioConfigId = planState.currentConfigId;
-        this.loadConfig();
+        if (this.scenarioConfigId) {
```

Comment:
Where does `this.scenarioConfigId` gets updated? not sure if I understand this change after we get the plan state